api-docs/cloud-serverless/ref.yml

```diff
@@ -13977,25 +13986,43 @@ paths:
           - [Best practices for optimizing writes](/influxdb/cloud-serverless/write-data/best-practices/optimize-writes/)
         required: true
       responses:
-        '204':
+        '201':
           description: |
-            Success. Data is written and queryable.
+            Created. Data in the batch is written and queryable.
+
+            If some points in the batch are rejected, the response body contains details about the [rejected points](/influxdb/cloud-serverless/write-data/troubleshoot/#troubleshoot-rejected-points), up to 100 points.
+          content:
+            application/json:
+              examples:
+                partialWriteErrorWithRejectedPoints:
+                  summary: Partial write rejects points with syntax errors
+                  value:
```

Comment:
@Jayclifford345 `201` (partial write) and `400` (batch failed) response body properties.